src/earn/types.ts

```diff
@@ -64,4 +65,14 @@ export interface BeforeDepositAction {
   onPress: () => void
 }
 
+export interface WithdrawAction {
+  name: WithdrawActionName
+  title: string
+  details: string
+  iconComponent: React.MemoExoticComponent<({ color }: { color: Colors }) => JSX.Element>
+  onPress: () => void
+}
+
 export type EarnEnterMode = 'deposit' | 'swap-deposit' | 'withdraw'
+
+export type EarnWithdrawType = 'partialWithdraw' | 'claim' | 'exit'
```

Comment:
`EarnWithdrawType` & `WithdrawActionName` look similar is there a reason not to combine them?

Reply:
good catch! updated in [3f4abb2](https://github.com/valora-inc/wallet/pull/6142/commits/3f4abb259406ae5cfedcf8b8c65778c428b5baed)